athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/data/BlockTest.java

```diff
@@ -975,7 +975,8 @@ private void assertSerializationOverhead(ByteArrayOutputStream serializedBlock)
         try {
             ByteArrayOutputStream jout = new ByteArrayOutputStream();
             JsonFactory factory = new JsonFactory();
-            JsonGenerator jsonGenerator = factory.createJsonGenerator(jout);
+            JsonGenerator jsonGenerator = factory.createGenerator(jout);
+            jsonGenerator.writeStartObject();
```

Comment:
Why did this have to change?

Reply:
JsonGenerator was marked deprecated, so I updated the usage.